### ISSUE TITLE
[Accessibility]A11y__AzureDataStudio_CreateExternalTable_ScreenReader:Voice Over is not announcing status while page is loading state.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#6071]
[Check out Keros!](https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags: #A11yMAS; #A11yTCS; #MAS1.3.1; #A11y_AzureDataStudio; #Win32; #DesktopApp

**Environment Details:**
Application name: Azure Data Studio
Version: 1.9.0-insider (user setup)
Commit: 2ca39b5
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS: MacOS Mojave 10.14.5
**Additional Details:**
Tools Name: Voice Over
MAS Number :MAS1.3.1
**Steps to Reproduce** :
1.Install the app Azure Data Studio.
2.Enable SQL server2019 preview->select table from the database.
3.Start Voice Over
4.Observe that while page is loading state ,screen reader doesn't announce "Page is still loading".
**Actual** :
When page is in loading state ,screen reader is not announcing "Page is still loading ".It only announces the information which present on that page.
**Expected**:
Screen reader should announce "Page is still loading " when page is loading.
**Recommendation**:
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx
**User Impact**:
Screen reader only user won't understand that page is loading and may misunderstood that page does not contain any content.
**MAS Reference**:
**MAS1.3.1**-
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}
**Attachment for Reference**:
[6071_[Accessibility]A11y__AzureDataStudio_CreateExternalTable_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3300921/6071_.Accessibility.A11y__AzureDataStudio_CreateExternalTable_ScreenReader.pptx)
Does this issue occur when all extensions are disabled?: Yes

**Comment by Charles-Gagnon** [comment]
@AccessibilityTestingTeam-TCS - What's the expected behavior when there are sections of the page loaded at the start but some are still loading? Should we always announce page loading when entering the page? And then what should we do when the page is fully loaded - announce something like "Page loaded"?

Or should we only announce loading for the components if they're selected (i.e. if we tab into/mouse over the loading component)?
